crates/sui-framework/sources/governance/validator_set.move

```diff
@@ -535,7 +535,7 @@ module sui::validator_set {
     /// is removed from `validators` and its staking pool is put into the `inactive_validators` table.
     fun process_pending_removals(
         self: &mut ValidatorSet,
-        ctx: &mut TxContext,
+        new_epoch: u64,
```

Comment:
Personally I think it's cleaner to just recompute new_epoch in process_pending_removals

Reply:
Sure. Will change it